client/modules/core/main.js

```diff
@@ -196,6 +196,17 @@ export default {
     return this.hasPermission(dashboardPermissions);
   },
 
+  getSellerShopId: function (userId = Meteor.userId()) {
+    if (userId) {
+      const group = Roles.getGroupsForUser(userId, "admin")[0];
+      if (group) {
+        return group;
+      }
+    }
+
+    return this.getShopId();
+  },
```

Comment:
this means for now, there can only be one shop per user, which I think a good start ( just something to document.  I can easily see having multiple shops per user, as long as we keep getShopId() as the source of truth that should be easy enough to do.